applications/n8n/.terraform.lock.hcl

```diff
@@ -0,0 +1,60 @@
+# This file is maintained automatically by "terraform init".
```

Comment:
I think we can delete this file for now. Since terraform init will generate a new one for it